package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cht-conf-test-harness",
-  "version": "2.4.0",
```

Comment:
not sure how this happened but `2.4.1` is already live https://www.npmjs.com/package/cht-conf-test-harness . would you `npm version patch` one more time and then commit to master?

Reply:
done!